pkg/features/kube_features.go

```diff
@@ -470,7 +470,7 @@ var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureS
 	RunAsGroup:                                  {Default: false, PreRelease: utilfeature.Alpha},
 	VolumeSubpath:                               {Default: true, PreRelease: utilfeature.GA},
 	BalanceAttachedNodeVolumes:                  {Default: false, PreRelease: utilfeature.Alpha},
-	PodReadinessGates:                           {Default: true, PreRelease: utilfeature.Beta},
+	PodReadinessGates:                           {Default: true, PreRelease: utilfeature.GA},
```

Comment:
typically, when promoting a feature gate to GA, we should:
* set `LockToDefault: true`
* add a comment to delete in 2 releases (per https://kubernetes.io/docs/reference/using-api/deprecation-policy/#deprecating-a-feature-or-behavior, rule 8)
* add a release note that the feature is now enabled, the feature gate will not allow disabling it, and the feature gate will be removed in 1.16

Reply:
Thanks! I have made the corresponding changes.